src/components/reusables/ActionBar/ActionBar.tsx

```diff
@@ -1,63 +1,110 @@
-import { Button, ButtonColor, ButtonVariant } from '@digdir/design-system-react';
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
 import * as React from 'react';
+import { useId } from 'react';
+import cn from 'classnames';
 
-import { ReactComponent as MinusCircle } from '@/assets/MinusCircle.svg';
-import { ReactComponent as AddCircle } from '@/assets/AddCircle.svg';
-
+import type { ClickHandler } from './Context';
+import { ActionBarContext } from './Context';
 import classes from './ActionBar.module.css';
+import { ActionBarIcon } from './ActionBarIcon';
+import { ActionBarContent } from './ActionBarContent';
 
 export interface ActionBarProps {
-  title: string;
-  subtitle?: string;
-  icon: ActionIconVariant;
-  actionCallBack: () => void;
-}
-
-export enum ActionIconVariant {
-  Add,
-  Remove,
+  actions?: React.ReactNode;
+  additionalText?: React.ReactNode;
+  children?: React.ReactNode;
+  color?: 'light' | 'neutral' | 'warning' | 'success' | 'danger';
+  iconVariant?: 'primary' | 'secondary';
+  onClick?: ClickHandler;
+  open?: boolean;
+  subtitle?: React.ReactNode;
+  title?: React.ReactNode;
 }
 
 export const ActionBar = ({
-  title,
+  actions,
+  additionalText,
```

Comment:
I just realized that additionalText is in fact not a string, but instead a ReactNode. So that makes for another question: What is then the difference between the actions and the additionalText when I could just as easily send a button to the AdditionalText prop and a text to the action prop?

Reply:
additionalText will always be placed in the middle between Title and actions. If we delete additionalText we will have no control over where it's placed it can be placed in the middle or on the end. Although it's a ReactNode you're not bound to sending in that, you could send in a string as well. Which I think is much easier than wrapping something in a <> and a div and along with a button, like it would be if actions was supposed to contain additionalText and action buttons.

Also I think the prop-name actions only make sense when you give buttons to it, not when you're suppsoed to give non-functional stuff like strings to it.

Reply:
I agree but we won't be able to have a line through and style the text like we want if we set it to be a string